lab_5_scrapper/scrapper.py

```diff
@@ -100,6 +229,14 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    sleep(randrange(3))
+
+    headers = config.get_headers()
+    timeout = config.get_timeout()
+
+    response = requests.get(url=url, headers=headers, timeout=timeout)
```

Comment:
return immediately, stop making intermediate variables

Reply:
Fixed it here and where was possible